src/test/java/hudson/plugins/active_directory/ActiveDirectoryDomainFipsEnabledTest.java

```diff
@@ -0,0 +1,63 @@
+package hudson.plugins.active_directory;
+
+import java.nio.file.Paths;
+
+import hudson.ExtensionList;
+import hudson.util.FormValidation;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestRule;
+import org.jvnet.hudson.test.FlagRule;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import io.jenkins.plugins.casc.ConfigurationAsCode;
+import io.jenkins.plugins.casc.ConfiguratorException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+import static org.junit.Assert.fail;
+
+public class ActiveDirectoryDomainFipsEnabledTest {
+
```

Comment:
I would like to see some form of configRoundtrip test as well, imo just calling a constructor is not enough. Constructors change and gets deprecated over time and this would then just test dead code. Bu testing the form submission itself fails we make sure that it sticks around.